modules/installation-gcp-tested-machine-types.adoc

```diff
@@ -15,6 +15,13 @@
 
 The following Google Cloud Platform instance types have been tested with {product-title}.
 
+[NOTE]
+====
+Not all instance types are available in all regions and zones. For a detailed breakdown of which instance types are available in which zones, see Google's documentation about link:https://cloud.google.com/compute/docs/regions-zones#available[regions and zones].
```

Comment:
```suggestion
Not all instance types are available in all regions and zones. For a detailed breakdown of which instance types are available in which zones, see link:https://cloud.google.com/compute/docs/regions-zones#available[regions and zones] (Google documentation).
```
Guideline about an external documentation:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute
> When pointing to a link that exists external to Red Hat’s customer portal documentation, add the link attribute, and use the <document_heading_name> (<document_source>) format.